.github/workflows/release.yml

```diff
@@ -224,15 +226,12 @@ jobs:
     name: Upload release asset to m.mautic.org
     needs: [release, test-fresh-install, test-update-install]
     # We only want this job to run in Mautic's repo, not in forks
-    if: github.repository_owner == 'mautic' && env.IS_PRERELEASE != true
+    if: github.repository_owner == 'mautic' && needs.release.outputs.is-prerelease
```

Comment:
This is the most important fix for the issue that was blocking the pipeline before: 

`The workflow is not valid. .github/workflows/release.yml (Line: 227, Col: 9): Unrecognized named-value: 'env'. Located at position 40 within expression: github.repository_owner == 'mautic' && env.IS_PRERELEASE != true`